spec.md

```diff
@@ -717,7 +717,7 @@ message ControllerPublishVolumeResponse {
   // The SP specific information that will be passed to the Plugin in
   // the subsequent `NodePublishVolume` call for the given volume.
   // This information is opaque to the CO. This field is OPTIONAL.
-  map<string, string> publish_volume_info = 1;
+  map<string, string> publish_volume = 1;
```

Comment:
I would actually rename this to `publish_info`. `publish_volume` sounds a bit weird.

Reply:
Yep, +1 for publish_info, done